hat/app/org/hatdex/hat/api/controllers/RichData.scala

```diff
@@ -202,6 +202,19 @@ class RichData @Inject() (
     } getOrElse Set()
 
   // Extract to namespace library
+  private def authorizeEndpointDataRead(
+      namespace: String,
+      appStatus: Option[HatApplication]
+    )(implicit
+      user: HatUser,
+      authenticator: HatApiAuthEnvironment#A): Boolean = {
+
+    val role = NamespaceRead(namespace)
+    WithRole.isAuthorized(user, authenticator, role) || appStatus.exists(
+      ContainsApplicationRole.isAuthorized(user, _, authenticator, role)
```

Comment:
Does this work for Owner as well?
Correct me if I'm wrong, but owner doesn't have NamespaceRead permission and needs to have an extra check for this case.

Reply:
this method is not used... I need to remove it